### PR TITLE
H2O python sample apps

### PR DESCRIPTION
**Pull request by danielfrg**
Adding two sample app using H2O python, including deploying a MOJO model from Python.

### CODE REVIEW

File: algo-dev-demo/h2o-python-mojo/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+In this demo we'll go through deploying a pre-trained [H2O](https://www.h2o.ai) model for insurance claims predictions.
+This model is saved as a MOJO (Model Object, Optimized).
+For more information on these objects see the [H2O documentation](http://docs.h2o.ai/h2o/latest-stable/h2o-docs/mojo-quickstart.html).
+
+This whole demo uses the [H2O python library](http://h2o-release.s3.amazonaws.com) to train and make prediction of the model.
+The model was trained using the `train.py` script located on this directory but this MOJO model
+could also be trained, saved and served using other H2O supported languages.
+
+This demo features the usage of an special environment designed to run H2O models in algorithmia.
```
Comment (kennydaniel) [inline algo-dev-demo/h2o-python-mojo/README.md:11]: Capitalize Algorithmia please

File: algo-dev-demo/h2o-python-mojo/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Other examples with H2O:
+- Deploying H2O models (Python)
```
Comment (kennydaniel) [inline algo-dev-demo/h2o-python-mojo/README.md:14]: Should this be a link?

File: algo-dev-demo/h2o-python-mojo/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+## Upload pre-trained model to Data Collections
+
+We are going to host the serialized pre-trained model on the Algorithmia platform, for more info look at the [Data Collections docs](https://algorithmia.com/developers/data/hosted).
+
+To create a Data Collection:
+
+- Log in to your Algorithmia account and click on **Data Sources** on the left navigation sidebar
+- Click on **New Data Source** on the top right corner, select **Hosted Data Collection** and name it `h2o_demo`
+- You can now select or drag and drop the pre-trained model: [data/DeepLearning_model_python_1611090304383_1.zip](https://github.com/algorithmiaio/sample-apps/raw/master/algo-dev-demo/h2o-python-mojo/data/DeepLearning_model_python_1611090304383_1.zip) to upload it to the Algorithmia Hosted Collection
+- Once the file it's uploaded you can copy the URI by clicking the triple dots to the right of the file name (e.g. `data://<USERNAME>/h2o_demo/DeepLearning_model_python_1611090304383_1.zip`)
```
Comment (kennydaniel) [inline algo-dev-demo/h2o-python-mojo/README.md:29]: "Once the file **is** uploaded**,** you can"

File: algo-dev-demo/h2o-python-mojo/README.md (hunk 2 above)
Comment (danielfrg) [reply algo-dev-demo/h2o-python-mojo/README.md]: Yes, done.

File: algo-dev-demo/h2o-python-mojo/README.md (hunk 1 above)
Comment (danielfrg) [reply algo-dev-demo/h2o-python-mojo/README.md]: Done

File: algo-dev-demo/h2o-python/README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+## Upload pre-trained model to Data Collections
+
+We are going to host the serialized pre-trained model on the Algorithmia platform, for more info look at the [Data Collections docs](https://algorithmia.com/developers/data/hosted).
+
+To create a Data Collection:
+
+- Log in to your Algorithmia account and click on **Data Sources** on the left navigation sidebar
+- Click on **New Data Source** on the top right corner, select **Hosted Data Collection** and name it `h2o_demo`
+- You can now select or drag and drop the pre-trained model: [data/DeepLearning_model_python_1610746052547_1](https://github.com/algorithmiaio/sample-apps/raw/master/algo-dev-demo/h2o-python/data/DeepLearning_model_python_1610746052547_1) to upload it to the Algorithmia Hosted Collection
+- Once the file it's uploaded you can copy the URI by clicking the triple dots to the right of the file name (e.g. `data://<USERNAME>/h2o_demo/DeepLearning_model_python_1610746052547_1`)
```
Comment (kennydaniel) [inline algo-dev-demo/h2o-python/README.md:27]: same fixes as above

File: algo-dev-demo/h2o-python-mojo/README.md (hunk 3 above)
Comment (danielfrg) [reply algo-dev-demo/h2o-python-mojo/README.md]: Done

File: algo-dev-demo/h2o-python/README.md (hunk 4 above)
Comment (danielfrg) [reply algo-dev-demo/h2o-python/README.md]: Done

**Review by kennydaniel — APPROVED** [review]

**Review by koverholt — APPROVED** [review]
LGTM, thank you for these examples!

**Pull request merged**